Makefile

```diff
@@ -8,6 +7,31 @@ DOCS_ROOT := docs
 API_TESTS_ROOT := api-tests
 CONVEYOR_SCHEMA := conveyor_schema.json
 
+# user data and indexes
+USER_DATA_ROOT := ./user-data
+
+# user accounts
+USER_ACCTS_ROOT := $(USER_DATA_ROOT)/accounts
+USER_ACCTS_FILE := $(USER_ACCTS_ROOT)/accounts.js
+
+# bad actor / GDPR user lists
+LISTS_ROOT := $(USER_DATA_ROOT)/lists
+GDPR_LISTS := raw.githubusercontent.com/steemit/condenser/master/src/app/utils/GDPRUserList.js
+LOCAL_GDPR_LISTS := $(addprefix $(LISTS_ROOT)/gdpr/srcs/, $(GDPR_LISTS))
+
+BAD_ACTOR_LISTS := raw.githubusercontent.com/steemit/condenser/master/src/app/utils/DMCAUserList.js \
```

Comment:
Could you add quick details to README about how these lists are kept up to date and built?